web/ui/module/lezer-promql/src/promql.grammar

```diff
@@ -388,8 +390,10 @@ NumberLiteral  {
   Deriv { condFn<"deriv"> }
   Exp { condFn<"exp"> }
   Floor { condFn<"floor"> }
+  HistogramCount { condFn<"histogram_count"> }
   HistogramFraction { condFn<"histogram_fraction"> }
   HistogramQuantile { condFn<"histogram_quantile"> }
+  HistogramSum { condFn<"histogram_sum"> }
```

Comment:
could you complete the file [highlight.js](https://github.com/prometheus/prometheus/blob/main/web/ui/module/lezer-promql/src/highlight.js#L22) ?
It contains also a list of function that will be used for the highlight.

Reply:
probably you will want to add the function `HistogramFraction` too

Reply:
Indeed. Thanks for the catch. Fixed in the last commit.

(I have no clue what I'm doing here. I just try to recognize the patterns… ;)

Reply:
Or I'm just very good in taking hard guesses…

Reply:
Indeed you are right :)